UI/Reports/filters/inventory_adj.html

```diff
@@ -1,5 +1,4 @@
 <?lsmb
```

Comment:
Every html file but this one has the process aligned on the same line as `lsmb`. Should we stay coherent on style?